db/data/20211202100614_correct_hpitt_science_subjects.rb

```diff
@@ -0,0 +1,301 @@
+      [9359, "chemistry"],
+      [9951, "biology"],
+      [9460, "biology"],
+      [9712, "biology"],
+      [9069, "biology"],
+      [9153, "biology"],
+      [9173, "biology"],
+      [9243, "chemistry"],
+      [10056, "biology"],
+      [9995, "biology"],
+      [9895, "biology"],
+      [9140, "chemistry"],
+      [10115, "biology"],
+      [9520, "biology"],
+      [9457, "biology"],
+      [9671, "physics"],
+      [8845, "biology"],
+      [10195, "chemistry"],
+      [9889, "chemistry"],
+      [10199, "biology"],
+      [9834, "chemistry"],
+      [9221, "biology"],
+      [8980, "biology"],
+      [9473, "biology"],
+      [9475, "biology"],
+      [9367, "chemistry"],
+      [9184, "biology"],
+      [9570, "physics"],
+      [9561, "biology"],
+      [10163, "biology"],
+      [9868, "biology"],
+      [10057, "chemistry"],
+      [8844, "biology"],
+      [9446, "chemistry"],
+      [9755, "physics"],
+      [9402, "biology"],
+      [9523, "physics"],
+      [10096, "biology"],
+      [10190, "biology"],
+      [10188, "physics"],
+      [8859, "biology"],
+      [9465, "physics"],
+      [8988, "biology"],
+      [9144, "biology"],
+      [8929, "biology"],
+      [10043, "biology"],
+      [9217, "biology"],
+      [10320, "chemistry"],
+      [10226, "biology"],
+      [9158, "biology"],
+      [8824, "biology"],
+      [9568, "biology"],
+      [9412, "biology"],
+      [9621, "physics"],
+      [9651, "chemistry"],
+      [9787, "biology"],
+      [9592, "chemistry"],
+      [9066, "biology"],
+      [9106, "biology"],
+      [9915, "biology"],
+      [8864, "physics"],
+      [9094, "chemistry"],
+      [9973, "chemistry"],
+      [10256, "biology"],
+      [9640, "chemistry"],
+      [10137, "chemistry"],
+      [9281, "physics"],
+      [9159, "chemistry"],
+      [8916, "biology"],
+      [9395, "biology"],
+      [10033, "biology"],
+      [8987, "biology"],
+      [8967, "biology"],
+      [9909, "biology"],
+      [9329, "biology"],
+      [9102, "biology"],
+      [9706, "biology"],
+      [9937, "biology"],
+      [8981, "biology"],
+      [10130, "biology"],
+      [8800, "physics"],
+      [9350, "chemistry"],
+      [10136, "biology"],
+      [8927, "biology"],
+      [9424, "biology"],
+      [10228, "biology"],
+      [8801, "biology"],
+      [8781, "biology"],
+      [9980, "chemistry"],
+      [9664, "chemistry"],
+      [9298, "biology"],
+      [9223, "chemistry"],
+      [8919, "biology"],
+      [8839, "biology"],
+      [9480, "biology"],
+      [9494, "biology"],
+      [9135, "biology"],
+      [10183, "biology"],
+      [9006, "biology"],
+      [9813, "biology"],
+      [8922, "biology"],
+      [8926, "physics"],
+      [9325, "biology"],
+      [10149, "chemistry"],
+      [9732, "chemistry"],
+      [10067, "chemistry"],
+      [9277, "chemistry"],
+      [9360, "physics"],
+    ]
+
+    correct_subjects.each do |id, subject|
+      trainee = hpitt_trainee_scope.find(id)
```

Comment:
This will fail in non-prod environments.

```suggestion
      trainee = hpitt_trainee_scope.find(id)
      trainee&.update(course_subject_one: subject)
```

Reply:
Oh yes. So it will. I think the line before will raise as well. What an amateur 🤣 I'll sort that out. Thanks @jhenderson